napari/plugins/_skimage_data.py

```diff
@@ -0,0 +1,58 @@
+from napari_plugin_engine import napari_hook_implementation
```

Comment:
Oh man, this file points to terrible patterns in the skimage data model! :sweat_smile: :confounded: (specifically: you can't just get the dir of the data module, and we don't provide any metadata programmatically.) Nice work fixing our stuff. :facepalm:

Reply:
well... to be fair to scikit-image, it would be very hard to avoid manual curation here.  For instance, `download_all` is a very reasonable thing to export... but not reasonable here.  and it's probably worth having `cat` and `chelsea` for historical reasons, but maybe not here...